cli/tests/testdata/module_graph/file_tests-checkwithconfig.ts

```diff
@@ -1,4 +1,4 @@
-import { ServerRequest } from "https://deno.land/std/http/server.ts";
+import { ServerRequest } from "https://deno.land/std/http/server_legacy.ts";
```

Comment:
Yikes, this file should use `test_util/std/` instead of remote server.

Reply:
That’ll be why then 😄 was confused by the inconsistency 😅